server/conf/i18n/messages

```diff
@@ -25,6 +25,8 @@ footer.returnToTop=Return to top
 # A message in the footer directing users to technical support. The placeholder is a link to
 # send an email to CiviForm technical support.
 footer.technicalSupport=For technical support: {0}
+# A Message in the footer that links the user to the official website of the specified Civic entry
+footer.informationServices = Looking for more information services? {0}
```

Comment:
(blocking) Based on what I see in the screenshots for the issue there should be two new lines here instead of one.

- Keep `footer.informationServices`, but remove the '{0}' token.
- Add a new one that contains 'Visit {0}'